### PR TITLE
Updated Swedish translations

### PR DESCRIPTION
**Pull request by Abbe98**
Before merging this has to be tested(help wanted).

[Testing Translations](https://github.com/OpenOrienteering/mapper/wiki/Translation)

### CODE REVIEW

File: translations/OpenOrienteering_sv.ts [hunk 1]
```diff
@@ -438,12 +438,12 @@
     <message>
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>CMYK</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 2]
```diff
@@ -438,12 +438,12 @@
...
     </message>
     <message>
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>RGB</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 3]
```diff
@@ -3824,22 +3824,22 @@ Skala om den importerade datan?</translation>
...
         <location filename="../src/map_editor.cpp" line="679"/>
         <source>&amp;Image</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Bild</translation>
     </message>
     <message>
         <location filename="../src/map_editor.cpp" line="681"/>
         <source>&amp;PDF</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 4]
```diff
@@ -5114,7 +5118,7 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/util/item_delegates.cpp" line="147"/>
         <location filename="../src/util/item_delegates.cpp" line="154"/>
         <source>%</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 5]
```diff
@@ -5471,7 +5475,7 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/gui/print_widget.cpp" line="132"/>
         <location filename="../src/gui/print_widget.cpp" line="135"/>
         <source>mm</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 6]
```diff
@@ -5512,33 +5516,33 @@ zooma med scrollhjulet om tillgängligt.</translation>
...
         <location filename="../src/gui/print_widget.cpp" line="228"/>
         <source>Export...</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportera...</translation>
     </message>
     <message>
         <location filename="../src/gui/print_widget.cpp" line="332"/>
         <source>PDF export</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 7]
```diff
@@ -5512,33 +5516,33 @@ zooma med scrollhjulet om tillgängligt.</translation>
...
         <source>Save to PDF</source>
-        <translation type="unfinished"></translation>
+        <translation>Spara som PDF</translation>
     </message>
     <message>
         <location filename="../src/gui/print_widget.cpp" line="807"/>
         <location filename="../src/gui/print_widget.cpp" line="847"/>
         <source>dpi</source>
-        <translation type="unfinished">dpi</translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts [hunk 8]
```diff
@@ -5558,27 +5562,27 @@ zooma med scrollhjulet om tillgängligt.</translation>
     <message>
         <location filename="../src/gui/print_widget.cpp" line="1005"/>
         <source>PNG</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: A finished entry must not be empty.

File: translations/OpenOrienteering_sv.ts (hunk 1 above)
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: I need to test if empty entries result in empty or untranslated ouput. However, it is at least difficult to tell whether the missing entry is intentionally or by mistake.

Comment (Abbe98) [inline translations/OpenOrienteering_sv.ts]: It's by mistake, updating it right away...

Comment (Abbe98) [inline translations/OpenOrienteering_sv.ts]: Or looking into it it's not a mistake, sorry saw this from my phone.

The past translator did just leave the finished element empty when the
value should be equal to the source...

**Comment by dg0yt** [comment]
Okay, than it should be okay. I will add your name to the list of contributors when I merge this change. Thank you very much.

File: translations/OpenOrienteering_sv.ts [hunk 9]
```diff
@@ -6236,68 +6240,68 @@ Högsta versionen som stöds är %2.</translation>
...
         <source>mm</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
     </message>
     <message>
         <location filename="../src/map_dialog_rotate.cpp" line="59"/>
         <source>X:</source>
         <comment>x coordinate</comment>
-        <translation type="unfinished"></translation>
+        <translatio></translation>
```
Comment (dg0yt) [inline translations/OpenOrienteering_sv.ts]: This must be "translation", or Qt Linguist won't open the file.

**Comment by dg0yt** [comment]
There are 685 translations for 1216 strings in total. Shall I wait for further updates before merging? Note that I will also have to run another round of updates to the untranslated strings soon.

**Comment by Abbe98** [comment]
Will do some more work in the coming days.

**Comment by dg0yt** [comment]
Note: I updated all translations from the sources after this merge. So you will now find some new and changed source strings.

**Comment by Abbe98** [comment]
Woops, I have a translated most string but not yet committed, some
conflicts may appear now.

**Comment by dg0yt** [comment]
Don't mind, you can simply overwrite the file. Your version will have more translations, and the other parts get regenerated.

**Comment by Abbe98** [comment]
Great thanks then I make push before adding more strings.
